bin/pyntlm_auth/rpc.py

```diff
@@ -0,0 +1,156 @@
+import global_vars
+import config_generator
+from samba import param, NTSTATUSError, ntstatus
+from samba.credentials import Credentials, DONT_USE_KERBEROS
+from samba.dcerpc.misc import SEC_CHAN_WKSTA
+from samba.dcerpc import netlogon
+import utils
+import datetime
+from samba.dcerpc.netlogon import (netr_Authenticator, MSV1_0_ALLOW_WORKSTATION_TRUST_ACCOUNT, MSV1_0_ALLOW_MSVCHAPV2)
+import binascii
+
+
+def init_secure_connection():
+    netbios_name = global_vars.c_netbios_name
+    realm = global_vars.c_realm
+    server_string = global_vars.c_server_string
+    workgroup = global_vars.c_workgroup
+    workstation = global_vars.c_workstation
+    password = global_vars.c_password
+    domain = global_vars.c_domain
+    username = global_vars.c_username
+    server_name = global_vars.c_server_name
+
+    lp = param.LoadParm()
+    try:
+        config_generator.generate_empty_conf()
+        lp.load("/root/default.conf")
```

Comment:
Why /root ?

Reply:
moved from original app.py,
for persistence purpose. put it to somewhere like /usr/local/pf/var/conf ?